src/Components/Server/src/Circuits/ArrayBuilderMemoryStream.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Web.Rendering
+{
+    /// <summary>
+    /// Writeable memory stream backed by a an <see cref="ArrayBuilder{T}"/>.
+    /// </summary>
+    internal sealed class ArrayBuilderMemoryStream : Stream
```

Comment:
Why write comments for an internal class?

Reply:
I started off by copying it from elsewhere. But it doesn't hurt to have it